spec-main/api-touch-bar-spec.ts

```diff
@@ -47,32 +54,35 @@ describe('TouchBar module', () => {
 
     it('can be added to and removed from a window', () => {
       const label = new TouchBarLabel({ label: 'bar' })
-      const touchBar = new TouchBar({ items: [
-        new TouchBarButton({ label: 'foo', backgroundColor: '#F00', click: () => {} }),
-        new TouchBarButton({
-          icon: path.join(__dirname, 'fixtures', 'assets', 'logo.png'),
-          iconPosition: 'right',
-          click: () => {}
-        }),
-        new TouchBarColorPicker({ selectedColor: '#F00', change: () => {} }),
-        new TouchBarGroup({ items: new TouchBar({ items: [new TouchBarLabel({ label: 'hello' })] }) }),
-        label,
-        new TouchBarPopover({ items: new TouchBar({ items: [new TouchBarButton({ label: 'pop' })] }) }),
-        new TouchBarSlider({ label: 'slide', value: 5, minValue: 2, maxValue: 75, change: () => {} }),
-        new TouchBarSpacer({ size: 'large' }),
-        new TouchBarSegmentedControl({
-          segmentStyle: 'capsule',
-          segments: [{ label: 'baz', enabled: false }],
-          selectedIndex: 5
-        }),
-        new TouchBarSegmentedControl({ segments: [] }),
-        new TouchBarScrubber({
-          items: [{ label: 'foo' }, { label: 'bar' }, { label: 'baz' }],
-          selectedStyle: 'outline',
-          mode: 'fixed',
-          showArrowButtons: true
-        })
-      ] })
+      const touchBar = new TouchBar({
```

Comment:
I added a line of code to the spec, but it seems like formatting on save changed a lot of whitespace.

Let me know if this needs to be reverted.